one_fm/hiring/doctype/duty_commencement/duty_commencement.json

```diff
@@ -243,13 +247,15 @@
    "read_only": 1
   },
   {
+   "depends_on": "eval:doc.employment_type!=\"Contract\"",
    "fetch_from": "job_offer.base",
    "fieldname": "base_salary",
    "fieldtype": "Currency",
    "label": "Base Salary",
    "read_only": 1
   },
   {
+   "depends_on": "eval:doc.employment_type!=\"Contract\" || doc.select_existing_salary_structure == 1",
```

Comment:
Hard coding with name field is not good. Any system manager can create or add the Employment type, since it is a Master doc